### PR TITLE
Add containers-sugar package

### PR DESCRIPTION
**Pull request by felipepaes**
This PR adds containers-sugar package recipe

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xmnlab** [comment]
@felipepaes please skip windows for now
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#skipping-builds
(`skip: True  # [win]`)

**Comment by xmnlab** [comment]
thanks for adding me here as a maintainer!

**Comment by xmnlab** [comment]
@felipepaes you need to add the `home` entry in the `about` section, for example: https://github.com/conda-forge/scicookie-feedstock/blob/main/recipe/meta.yaml#L41

File: recipes/containers-sugar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/containers_sugar-{{ version }}.tar.gz
+  sha256: 28c9ed69d61a52016065af927f00eea147eb148ff383c313543da1e9514a15c6
+
+build:
+  noarch: python
```
Comment (xmnlab) [inline recipes/containers-sugar/meta.yaml:13]:
```suggestion
```

This is something that I always have problems in the conda recipe.

but normally if there is a skip, you cannot use noarch ... but if you remove noarch, you need to remove the pinning for python?? ... I always struggles with that XD

but for now just remove the noarch entry

https://github.com/conda-forge/staged-recipes/pull/24383#issuecomment-1787034899

**Comment by felipepaes** [comment]
You were absolutely right @xmnlab we can pin python's version without the `noarch:python` xD

**Comment by felipepaes** [comment]
@conda-forge/help-python, ready for review!

**Comment by xmnlab** [comment]
@conda-forge/help-python , a friendly reminder about this PR, thank you :)

File: recipes/containers-sugar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/containers_sugar-{{ version }}.tar.gz
+  sha256: 28c9ed69d61a52016065af927f00eea147eb148ff383c313543da1e9514a15c6
+
+build:
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/containers-sugar/meta.yaml:13]: Same comment as https://github.com/conda-forge/staged-recipes/pull/24382/files#r1384945123

Comment (xmnlab) [reply recipes/containers-sugar/meta.yaml:13]: @ocefpaf thank you for the review!

question: containers-sugar works on top of docker-compose/docker .. does it work on windows?
I checked the docker-compose feedstock and I am very surprised that it doesn't have any skip for windows :O
do you know if I would need to have wsl to test that on windows?

Comment (ocefpaf) [reply recipes/containers-sugar/meta.yaml:13]: Maybe it is better to use the OS-noarch then. With the __unix virtual package.

Comment (xmnlab) [reply recipes/containers-sugar/meta.yaml:13]: got it .. thank you so much! appreciate that!

**Comment by felipepaes** [comment]
Thank you so much for your review @ocefpaf !  
Thanks for pointing us to the right track! 🙏

**Pull request merged**